app/services/select_no_uploads_strategy.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+# A strategy class that selects no uploads: a null strategy, if you will. Used
+# by the `DirectUploadsSweeper`
+class SelectNoUploadsStrategy
+  # @return [ActiveRecord::Relation]
+  def self.select
+    new.select
+  end
+
+  # @return [ActiveRecord::Relation]
+  def select
+    active_record_class.none
+  end
+
+  private
+
+  def active_record_class
+    @active_record_class ||= ActiveStorage::Blob
```

Comment:
why an ivar here? There's no way to set this to anything else. Can this just return a const?

Reply:
Not needed, will make the change you suggest.

Reply:
Done.